src/vos/tests/evt_ctl.sh

```diff
@@ -159,9 +159,9 @@ echo "$cmd"
 $cmd -t
 
 if [[ -z ${CMOCKA_MESSAGE_OUTPUT+x} || -z ${CMOCKA_XML_FILE+x} ]]; then
-	result=$?
+	result="${PIPESTATUS[0]}"
```

Comment:
Is this supposed to be the result of the `$cmd -t` on line 159?  If so, then it won't work.  The statements on line 161 will have overwritten the `${PIPESTATUS}` (and `$?`) variables.

Reply:
Yes, it is the value of $cmd -t. I can remove the result value out of the if condition.

Reply:
```bash
$ false; if [[ true || true ]]; then echo ${PIPESTATUS[0]}; fi
0
```
shows that `${PIPESTATUS[0]}` doesn't have the exit value of the `false` command before the `if` statement but rather is the exit status of the `if` statement.  You need to move the evaluation (or storing of the value) of `${PIPESTATUS[0]}` up so that it's immediately after the `$cmd -t`.

Reply:
I believe I changed this on the latest patch.